data_steward/cdr_cleaner/cleaning_rules/generate_derived_tables.py

```diff
@@ -0,0 +1,379 @@
+"""
+Create Derived Tables
+
+Until now, 'observation_period', 'drug_era, 'condition_era' were populated Pre-deid stages.
+But re-creating these tables post-deid will allow OHDSI tools to run on workbench more accurately.
+"""
+# Python imports
+import logging
+from datetime import datetime
+
+# Third party imports
+
+# Project imports
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+from resources import validate_date_string
+from common import CONDITION_ERA, DRUG_ERA, OBSERVATION_PERIOD, JINJA_ENV
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+
+LOGGER = logging.getLogger(__name__)
+
+DELETION_QUERY = JINJA_ENV.from_string("""
+DELETE FROM `{{project_id}}.{{dataset_id}}.{{storage_table_name}}`
+WHERE TRUE
+""")
+
+POPULATE_OBS_PRD = JINJA_ENV.from_string("""
+INSERT INTO `{{project_id}}.{{dataset_id}}.{{storage_table_name}}`
```

Comment:
Add cols for all 3 queries
```suggestion
INSERT INTO `{{project_id}}.{{dataset_id}}.{{storage_table_name}}` {{cols}}
```